etna/whatson/models.py

```diff
@@ -0,0 +1,578 @@
+    )
+
+    contact_info = RichTextField(
+        verbose_name=_("contact info"),
+        blank=True,
+        help_text=_("Information about who to contact regarding the event."),
+    )
+
+    # Promote tab
+    short_title = models.CharField(
+        max_length=50,
+        verbose_name=_("short title"),
+        help_text=_(
+            "A short title for the event. This will be used in the event listings."
+        ),
+    )
+
+    # DataLayerMixin overrides
+    gtm_content_group = "What's On"
+
+    class Meta:
+        verbose_name = _("event page")
+
+    content_panels = BasePageWithIntro.content_panels + [
+        FieldPanel("lead_image"),
+        MultiFieldPanel(
+            [
+                FieldPanel("event_type"),
+                FieldPanel("start_date", read_only=True),
+                FieldPanel("end_date", read_only=True),
+                InlinePanel(
+                    "sessions",
+                    heading=_("event sessions"),
+                    help_text=_("List of event sessions"),
+                    min_num=1,
+                ),
+                FieldPanel("description"),
+                FieldPanel("useful_info"),
+                FieldPanel("target_audience"),
+                InlinePanel(
+                    "event_access_types",
+                    heading=_("access types"),
+                    help_text=_(
+                        "If the event has more than one access type, please add these in order of relevance from most to least."
+                    ),
+                ),
+                InlinePanel(
+                    "event_audience_types",
+                    heading=_("audience types"),
+                    help_text=_(
+                        "If the event has more than one audience type, please add these in order of relevance from most to least."
+                    ),
+                ),
+                InlinePanel(
+                    "hosts",
+                    heading=_("host information"),
+                    help_text=_(
+                        "If the event has more than one host, please add these in order of relevance from most to least."
+                    ),
+                ),
+                InlinePanel(
+                    "speakers",
+                    heading=_("speaker information"),
+                    help_text=_(
+                        "If the event has more than one speaker, please add these in order of relevance from most to least."
+                    ),
+                ),
+            ],
+            heading=_("event information"),
+        ),
+        MultiFieldPanel(
+            [
+                FieldPanel("venue_type"),
+                FieldPanel("venue_website"),
+                FieldPanel("venue_address"),
+                FieldPanel("venue_space_name"),
+                FieldPanel("video_conference_info"),
+            ],
+            heading=_("venue information"),
```

Comment:
Wagtail doesn't automatically capitalise the heading unfortunately, so this should probably be:
```suggestion
            heading=_("Venue information"),
```